src/main/scala/mesosphere/marathon/core/launchqueue/impl/RateLimiter.scala

```diff
@@ -19,13 +19,21 @@ private[launchqueue] class RateLimiter(clock: Clock) {
   /** The task launch delays per app and their last config change. */
   private[this] var taskLaunchDelays = Map[(PathId, Timestamp), Delay]()
 
-  def cleanUpOverdueDelays(): Unit = {
+  def getMinimumTaskExecutionSeconds: Long = minimumTaskExecutionSeconds
+
+  /**
+    * Reset delay for tasks that have reached the viability
+    * threshold. The deadline indicates when the task has been
+    * launched for the last time.
+    */
+  def resetViableTasksDelays(): Unit = {
```

Comment:
`resetDelaysOfViableTasks` might be a better name, but I have no strong opinion about this.